src/peergos/shared/storage/ContentAddressedStorage.java

```diff
@@ -593,21 +593,17 @@ private static String encode(String component) {
 
         @Override
         public CompletableFuture<List<Multihash>> recursivePin(PublicKeyHash owner, Multihash hash) {
-            return poster.get(apiPrefix + PIN_ADD + "?stream-channels=true&arg=" + hash.toString()
-                    + "&owner=" + encode(owner.toString())).thenApply(this::getPins);
+            return Futures.of(Arrays.asList(hash));
```

Comment:
can you explain these changes?

Reply:
We're not calling pin on ipfs at all anymore. This was only relevant when we were using the built-in ipfs GC, but we haven't been doing that for a while now. Pinning stops data from being garbage collected by the ipfs GC.